sources/packages/web/src/components/common/students/applicationDetails/AssessmentAward.vue

```diff
@@ -33,6 +34,9 @@ export default defineComponent({
     confirmEnrolment: (disbursementId: number) => {
       return !!disbursementId;
     },
+    disbursementCancelled: (disbursementId: number) => {
```

Comment:
As mentioned in the PR description we can remove the disbursementId in the event as it is not used.